env.sh

```diff
@@ -31,23 +31,6 @@ if [[ -z ${DORIS_THIRDPARTY} ]]; then
     export DORIS_THIRDPARTY=${DORIS_HOME}/thirdparty
 fi
 
-# check java version
```

Comment:
we need to check Java and maven version here

Reply:
I have seen  the build script, the check java also in build.sh. Why you build jdk after check java in the build-thirdparty.sh

Reply:
I have seen the build script, the check java also in build.sh.  Why you build jdk after check java in the build-thirdparty.sh? And I have test the build after I delete this. You can test it in your environment.

Reply:
JAVA_HOME can be set by user in his environment, and not everyone use thirdparty's JDK as his JAVA_HOME.
So, we need to check java's version to make sure it can work with our project.